addons/point_of_sale/tests/test_frontend.py

```diff
@@ -888,19 +900,23 @@ def test_fiscal_position_inclusive_and_exclusive_tax(self):
             'amount_type': 'percent',
             'type_tax_use': 'sale',
         })
-        tax_inclusive_2 = self.env['account.tax'].create({
+        self.env['account.tax'].create({
             'name': 'Tax incl.10%',
             'amount': 10,
             'price_include_override': 'tax_included',
             'amount_type': 'percent',
             'type_tax_use': 'sale',
+            'fiscal_position_ids': fiscal_position_1 | fiscal_position_4,
```

Comment:
I saw that you changed a few, but there is a lot of places where you use recordsets instead of ORM commands, not listing them all